pkg/controller/secret/types.go

```diff
@@ -0,0 +1,36 @@
+package secret
+
+import (
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	AnnotationSecretGenerate    = "secret-generator.v1.mittwald.de/autogenerate"
+	AnnotationSecretGeneratedAt = "secret-generator.v1.mittwald.de/autogenerate-generated-at"
```

Comment:
Just a nitpick: IMO constant names and annotation strings should be the same:
```suggestion
	AnnotationSecretAutoGenerate    = "secret-generator.v1.mittwald.de/autogenerate"
	AnnotationSecretAutoGeneratedAt = "secret-generator.v1.mittwald.de/autogenerate-generated-at"
```